docs/using/codeless.rst

```diff
@@ -0,0 +1,204 @@
+
+.. code-block:: yaml
+
+  datagrabber:
+    kind: OasisVBMTesting
+
+
+Data Reader
+^^^^^^^^^^^
+
+As mentioned before, this section is entirely optional, as junifer only provides one data reader
+(:class:`junifer.datareader.DefaultDataReader`), which is the default in case the section is not specified.
+
+In any case, the syntax of the section is the same as for the ``datagrabber`` section, using the ``kind`` key to
+specify the data reader to use, and additional keys to pass parameters to the data reader:
+
+.. code-block:: yaml
+
+    datareader:
+      kind: DefaultDataReader
+
+
+For the *Oasis VBM Testing dataset* example, we will not specify a data reader step.
+
+Preprocessing
+^^^^^^^^^^^^^
+
+Preprocessing is also an optional step, as it might be the case that no pre-processing is needed. In the case that
+preprocessing is needed, the section must be configured using the ``kind`` key to specify the preprocessor to use, 
+and additional keys to pass parameters to the preprocessor.
+
+For example, to use the :class:`junifer.preprocess.fMRIPrepConfoundRemover` preprocessor, we just need to specify its
+name as the ``kind`` key, as well as its parameters.
+
+
+.. code-block:: yaml
+
+  preproces:
+    kind: fMRIPrepConfoundRemover
+    strategy:
+      motion: full
+      wm_csf: full
+      global_signal: basic
+    spike: 0.2
+    detrend: False
+    standardize: True
+
+
+For the *Oasis VBM Testing dataset* example, we will not specify a preprocessing step.
+
+
+Markers
+^^^^^^^
+
+The ``markers`` section diverges from the previous ones, as we need to specify a list of markers. Each marker has a
+name that we can use to refer to it later, and a set of parameters that will be passed to the marker.
+
+For the *Oasis VBM Testing dataset* example, we want to comute the mean ``VBM_GM`` value for each parcel using the 
+Schaefer parcellation (100 parcels, 7 networks), Schaefer parcellation (200 parcels, 7 networks), and the *DMNBuckner*
+network, using 5mm spheres. Thus, we will configure the ``markers`` section as follows:
+
+.. code-block:: yaml
+
+  markers:
+    - name: Schaefer100x7_mean
+      kind: ParcelAggregation
+      parcellation: Schaefer100x7
+      method: mean
+    - name: Schaefer200x7_mean
+      kind: ParcelAggregation
+      parcellation: Schaefer200x7
+      method: mean
+    - name: DMNBuckner_5mm_mean
+      kind: SphereAggregation
+      coords: DMNBuckner
+      radius: 5
+      method: mean
+
+
+Storage
+^^^^^^^
+
+Finally, we need to define how and where the results will be stored. This is done using the ``storage`` section,
+which must be configured using the ``kind`` key to specify the storage to use, and additional keys to pass parameters.
+
+For example, to use the :class:`junifer.storage.SQLiteFeatureStorage` storage, we just need to specify where we want
+to store the results:
+
+.. code-block:: yaml
+
+    storage:
+      kind: SQLiteFeatureStorage
+      uri: /data/junifer/example/oasis_vbm_testing.sqlite
```

Comment:
In the storage types, we have `.db` extension for SQLite. Just to be consistent, maybe we use it here?

Reply:
it does not matter, the user sets the name and extension. It should be sqlite.

Reply:
It indeed does not matter, my argument is just for the sake of consistency. I can imagine it be confusing for users who are not familiar with SQLite in that detail.